contrib/flo-bigquery/src/main/java/com/spotify/flo/contrib/bigquery/BigQueryOperation.java

```diff
@@ -86,27 +68,25 @@
     Provider() {
     }
 
-    public BigQueryOperation<T, Object> bq() {
+    public BigQueryOperation<T> bq() {
       return new BigQueryOperation<>();
     }
 
-    public BigQueryOperation<T, BigQueryResult> query(Fn<QueryRequest> queryRequest) {
-      return BigQueryOperation.ofQuery(queryRequest);
-    }
-
-    public BigQueryOperation<T, BigQueryResult> query(QueryRequest queryRequest) {
-      return BigQueryOperation.ofQuery(() -> queryRequest);
-    }
-
-    public BigQueryOperation<T, BigQueryResult> query(String query) {
-      return BigQueryOperation.ofQuery(() -> QueryRequest.of(query));
+    public BigQueryOperation<T> query(String query) {
+      final QueryJobConfiguration queryConfig = newBuilder(query)
+          // Use standard SQL syntax for queries.
```

Comment:
Should this be javadoc instead?

Reply:
Indeed. I just copied Google's sample code. xD